app/Models/Foundation/Summit/Summit.php

```diff
@@ -6868,4 +6875,47 @@ public function removeRegistrationFeedMetadata(SummitRegistrationFeedMetadata $m
         $metadata->clearSummit();
     }
 
+    /**
+     * @return SummitLeadReportSetting[]
+     */
+    public function getLeadReportSettings()
+    {
+        return $this->lead_report_settings;
+    }
+
+    /**
+     * @param SummitLeadReportSetting $lead_report_setting
+     */
+    public function addLeadReportSetting(SummitLeadReportSetting $lead_report_setting): void
+    {
+        $this->lead_report_settings->add($lead_report_setting);
```

Comment:
check here
 if($this->lead_report_settings->contains($lead_report_setting)) return;